[2024-03-12T14:34:02.690-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_data.validate_train manual__2024-03-12T18:33:26.408808+00:00 [queued]>
[2024-03-12T14:34:02.695-0400] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_data.validate_train manual__2024-03-12T18:33:26.408808+00:00 [queued]>
[2024-03-12T14:34:02.695-0400] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-03-12T14:34:02.715-0400] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): validate_train> on 2024-03-12 18:33:26.408808+00:00
[2024-03-12T14:34:02.721-0400] {standard_task_runner.py:60} INFO - Started process 341965 to run task
[2024-03-12T14:34:02.725-0400] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'train_data', 'validate_train', 'manual__2024-03-12T18:33:26.408808+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmp233xm7yc']
[2024-03-12T14:34:02.726-0400] {standard_task_runner.py:88} INFO - Job 47: Subtask validate_train
[2024-03-12T14:34:02.999-0400] {task_command.py:423} INFO - Running <TaskInstance: train_data.validate_train manual__2024-03-12T18:33:26.408808+00:00 [running]> on host DESKTOP-6U7QAQN.
[2024-03-12T14:34:03.055-0400] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='youremail@email.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_data' AIRFLOW_CTX_TASK_ID='validate_train' AIRFLOW_CTX_EXECUTION_DATE='2024-03-12T18:33:26.408808+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-12T18:33:26.408808+00:00'
[2024-03-12T14:34:03.334-0400] {logging_mixin.py:188} INFO - No anomalies detected in new data
[2024-03-12T14:34:03.335-0400] {python.py:202} INFO - Done. Returned value was: None
[2024-03-12T14:34:03.338-0400] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=train_data, task_id=validate_train, execution_date=20240312T183326, start_date=20240312T183402, end_date=20240312T183403
[2024-03-12T14:34:03.381-0400] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-12T14:34:03.562-0400] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
